### ISSUE TITLE
Problem with setup credentials

### ISSUE THREAD

**Issue by ricki158** [jbergler/hass-ttlock#151]
I successfully installed it on HACS, then created an account on the TTLock Open Platform. I'm now trying to add the integration, but it asks for a username and password. I entered the credentials from the TTLock Open Platform, but they didn’t work. I also tried using the credentials from the TTLock app (sciener.cn site), but it didn’t accept them either.

**Comment by mk28624** [comment]
Same for me.
Did you resoelve the issue?
![image](https://github.com/user-attachments/assets/747e6430-57b8-408e-8995-4b7b3cac6dea)

**Comment by roblencheg** [comment]
I have the same problem.(

**Comment by keldona** [comment]
Same problem as well.

**Comment by mk28624** [comment]
To doublecheck, which username and psswd should be used in the 'Login to TT Lock' window?
The same as in 'Open Platform'?
![1](https://github.com/user-attachments/assets/a85c59e8-44ef-4c75-9b6e-725a83ecdb65)
![image](https://github.com/user-attachments/assets/bf1169c5-9698-4a0e-8746-61e0352ae16a)

Also, I did successfully connected to lock first time, and it was working. Then I had to reset the addon and start it again. NOW it doesn't allow to tog in.
Did you had the same situation? - It was working for you 1st time but upon reset it doesn't anymore?
OR are you installing it for the first time? (meaning my resetting doesn't have any impact).

**Comment by krede21** [comment]
Same problem for me. Allowed to setup oAuth, but then prompted to login now just fails to log in.

**Comment by jbergler** [comment]
Hello - I've updated the readme to add some more info about the different credentials and how to handle some common cases. If that doesn't work, please start a discussion - bug reports aren't the right place for requesting support.

**Comment by mk28624** [comment]
Thank you, Jonas. Your updated readme helped.
All, SOLUTION: Delete old TT lock credentials from Application Credentials in HA, and start over.

**Comment by ricki158** [comment]
I tried using the application credentials (sciener.cn website) instead of those from https://euopen.ttlock.com/manager as indicated in the readme, but even so, I am not being accepted.

In the previous tab, however, I correctly entered the OAuth and password from the site.

I also deleted the credentials from HomeAssistant as indicated, to start with a clean installation, but I still couldn't get it to work.
